PowerEditor/src/WinControls/AboutDlg/AboutDlg.cpp

```diff
@@ -306,7 +306,11 @@ void DebugInfoDlg::doDialog()
 
 void DoSaveOrNotBox::doDialog(bool isRTL)
 {
-	
+	if (::IsIconic(_hParent))

```

Comment:
For unknown reason, ::IsIconic(_hParent) return always false under my Windows 10, even Notepad++ is iconized.
Have you tested your code under Windows 10 ?

Reply:
>Have you tested your code under Windows 10 ?

Yes @donho , I have tested. I verified on Win10 x64.